src/odemis/driver/test/simstreakcam_test.py

```diff
@@ -69,9 +68,6 @@ def setUp(self):
             "ignore", category=RuntimeWarning, message=re.escape("numpy.ndarray size changed")
         )
 
-    @classmethod
-    def setUpClass(cls):
-
         super(TestSimStreakCamGenericCam, cls).setUpClass()
```

Comment:
Typically, it's best to call super as the first line, unless there is good reason not too (I don't think there is).